docs/cse/administration/mitre-coverage.md

```diff
@@ -34,6 +35,9 @@ Watch this micro lesson to learn about the MITRE ATT&CK Threat Coverage Explorer
      allow="accelerometer; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
      allowfullscreen
      />
+-->
+
+<script src="https://fast.wistia.net/player.js" async></script><iframe src="https://fast.wistia.net/embed/iframe/yebz0v90tx?web_component=true&seo=true&videoFoam=false" title="Micro Lesson: Cloud SIEM MITRE ATT&amp;CK® Threat Coverage Explorer Video" allow="autoplay; fullscreen" allowtransparency="true" frameborder="0" scrolling="no" class="wistia_embed" name="wistia_embed" width="854px" height="480px"></iframe>
```

Comment:
Could you change to this format? This allows the video embedding to be responsive in desktop/mobile when users resize the window.

```suggestion
<Iframe url="https://fast.wistia.net/embed/iframe/yebz0v90tx?web_component=true&seo=true&videoFoam=false"
  width="854px"
  height="480px"
  id="wistiaVideo"
  className="video-container"
  display="initial"
  position="relative"
  allow="autoplay; fullscreen"
  allowfullscreen
/>
```

Here's what it looks like, not-responsive code vs responsive code.

https://github.com/user-attachments/assets/14576fa7-9560-4582-b3e2-eae2cfe95452

https://github.com/user-attachments/assets/aa8c08a6-1231-4279-8704-17c03857a64a